spec/graphql/schema/interface_spec.rb

```diff
@@ -39,8 +35,8 @@ def new_method
       assert_equal 2, new_object_2.fields.size
       # It got the new method
       assert new_object_2.method_defined?(:new_method)
-      # But not the old method
-      refute new_object_2.method_defined?(:id)
+      # And the inherited method
+      assert new_object_2.method_defined?(:id)
```

Comment:
(This was just a document-how-it-actually-works test, I think the new behavior is better anyway)